### DISCUSSION TITLE
Whitescreen Issues

### DISCUSSION THREAD

**Discussion by mazanyah** [iotaledger/firefly#1902] [Help & Support]
After providing stronghold backup file and correct password to restore Firefly Wallet I am getting a whitescreen and so cannot access my Firefly wallet for staking. Mac OS 10.13.6. 
Does anyone know how to fix this issue. Thanks.

**Comment by r-c-k** [comment]
Removing the Firefly folder from `~/Library/Application Support/Firefly` and then restoring your profile from your ledger/passphrase worked for some.

**Reply by mazanyah** [reply]
Thanks, but the use of the Ledger hardware wallet is not involved in this instance.

**Reply by r-c-k** [reply]
Then your passphrase it is from which you'd have to restore your profile.
